### PR TITLE
Documents desktop shortcut keys and how the compare to Jupyter front-ends

### PR DESCRIPTION
**Pull request by jameskoch**
@rgbkrk I mentioned I'd get this to you. I'm not actually pushing to get this merged unless you think it's useful. It's basically my notes from asking "are major shortcut key combos missing in Desktop?"

### CODE REVIEW

**Comment by rgbkrk** [comment]
Ooooh this is super great. I think this probably belongs in `docs/` somewhere. What do you think @captainsafia?

File: applications/desktop/SHORTCUT-KEYS.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+| |  |  |  |  |
+|notebook:change-cell-to-code | Y | Y | Ctrl+Shift+Y |  |
+|notebook:change-cell-to-markdown | M | M | Ctrl+Shift+M |  |
+|notebook:change-cell-to-raw | R | R |  |  |
+|notebook:change-to-cell-heading-1 | 1 | 1 |  | This changes the Markdown header level (###) |
+|notebook:change-to-cell-heading-2 | 2 | 2 |  |  |
+|notebook:change-to-cell-heading-3 | 3 | 3 |  |  |
+|notebook:change-to-cell-heading-4 | 4 | 4 |  |  |
+|notebook:change-to-cell-heading-5 | 5 | 5 |  |  |
+|notebook:change-to-cell-heading-6 | 6 | 6 |  |  |
```
Comment (rgbkrk) [inline applications/desktop/SHORTCUT-KEYS.md:37]: The change to cell heading hot keys I'm not a big fan of.

**Comment by rgbkrk** [comment]
Maybe `doc/development/keybinding-layout.md` ?

**Comment by captainsafia** [comment]
I think placing it on the `docs.nteract.io` website might work better. We can just copy this markdown file all over.

This way, this table will render in HTML and we can link to it through a Help > Keyboard Shortcuts menu item.

@jameskoch Adding this markdown file to https://github.com/nteract/docs should be (hopefully) painless. You'll need to create a `desktop/shortcut-keys.md` file and then modify `_sidebar.md` to create a link to that file.

**Comment by jameskoch** [comment]
Switched over the the docs repo - https://github.com/nteract/docs/pull/42

**Pull request closed without merging**